vars/enforceTrustedApproval.groovy

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/groovy
+
+/*
+ * To be able to use the method exposed here, the following script signatures
+ * must be approved:
+ *
+ * method org.kohsuke.github.GHUser isMemberOf org.kohsuke.github.GHOrganization
+ * method org.kohsuke.github.GitHub getOrganization java.lang.String
+ * method org.kohsuke.github.GitHub getUser java.lang.String
+ * method org.kohsuke.github.GitHubBuilder build
+ * method org.kohsuke.github.GitHubBuilder withOAuthToken java.lang.String java.lang.String
+ * new org.kohsuke.github.GitHubBuilder
+ *
+ */
+import org.kohsuke.github.GHOrganization
+import org.kohsuke.github.GHUser
+import org.kohsuke.github.GitHub
+import org.kohsuke.github.GitHubBuilder
+
+final boolean isOrgMember(String user, String org, gitHubCredentialsId) {
```

Comment:
Looks to be missing the type here 'String gitHubCredentialsId'

Reply:
Good spot, I'll update it